Ian Douglas assignment 6.Rmd

```diff
@@ -0,0 +1,48 @@
+---
+title: "Ian Douglas assignment 6"
+author: "ian douglas"
+date: "11/26/2018"
+output: html_document
+---
+```{R}
+M1 <- read.csv("MOOC1.csv", header = TRUE)
+
+M2 <- read.csv("MOOC2.csv", header = TRUE)
+```
+
+```{R}
+c.tree1 <- rpart(certified ~ forum.posts, data = M1)
+c.tree2 <- rpart(certified ~ grade, data = M1)
+c.tree3 <- rpart(certified ~ assignment, data = M1)
+c.tree4 <- rpart(certified ~ assignment + grade, data = M1)
+c.tree5 <- rpart(certified ~ assignment + forum.posts + grade, data = M1)
+printcp(c.tree1)
+printcp(c.tree2)
+printcp(c.tree3)
+printcp(c.tree4)
+printcp(c.tree5)
+```
+I conclude to keep only forum.posts in the tree because it is highly efficient, showing the lowest CP and relative error with only two branches.
```

Comment:
I'm not sure this is a result of the variable or because you have not scaled the variables first. `forum.posts` has many more increments than the other variables so will likely be preferenced by the algorithm.